src/Bicep.Cli/Commands/PublishCommand.cs

```diff
@@ -93,7 +93,7 @@ public async Task<int> RunAsync(PublishArguments args)
 
                 await ioContext.Output.WriteLineAsync(string.Format(CliResources.ExperimentalFeaturesDisclaimerMessage, PublishSourceFeatureName));
 
-                sourcesStream = SourceArchive.PackSourcesIntoStream(compilation.SourceFileGrouping, features.CacheRootDirectory);
+                sourcesStream = SourceArchive.PackSourcesIntoStream(moduleDispatcher, compilation.SourceFileGrouping, features.CacheRootDirectory);
```

Comment:
What is the motivation to return a stream and then transform into a `BinaryData` vs simply returing a `BinaryData` from this function?